src/utils/paging.ts

```diff
@@ -1,23 +1,22 @@
 import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
-import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
 import { pipe } from "fp-ts/lib/function";
 import * as O from "fp-ts/lib/Option";
 import * as t from "io-ts";
 
 export const PageResults = t.intersection([
   t.interface({
     hasMoreResults: t.boolean,
-    items: t.readonlyArray(t.interface({ id: NonEmptyString }))
+    items: t.readonlyArray(t.interface({ id: t.string }))
   }),
   t.partial({
-    next: NonEmptyString,
-    prev: NonEmptyString
+    next: t.string,
+    prev: t.string
```

Comment:
Why string? is `""` allowed?

Reply:
There are some older models types that have id defined as string. So we had an error passing them as type.

Reply:
Shouldn't we decode them?